projects/fal/tests/test_apps.py

```diff
@@ -1004,6 +1005,68 @@ def test_field_exception_billing(test_exception_app: AppClient):
         assert not hasattr(response.headers, "x-fal-billable-units")
 
 
+def test_stop_runner(host: api.FalServerlessHost, test_sleep_app: str):
+    def submit_and_wait_for_runner():
+        handle = apps.submit(test_sleep_app, arguments={"wait_time": 1})
+
```

Comment:
ah, one last thing. Lets move these tests (the ones you introduced) to `tests/api/test_runners.py` please while we are at it.